### ISSUE TITLE
[ACCESSIBILITY] The Status Card mode not available  

### ISSUE THREAD

**Issue by timo54** [rokwire/illinois-app#68]
**Describe the issue**
On the Status Card window, there are two views - one of the Building Access status, the other is the QR code to identify the user. These two views can be accessed with a two-finger horizontal swipe while using the screen reader but they are not announced. The two-buttons which illustrate which window you're on are not focusable via swiping and are not announced. The screen reader user will not know these two screens are available and will not know how to switch between them to show their QR code identity or their building access status. 

**To Reproduce**
1. Turn the screen reader on
2. Go to the Status Card
3. Status Card mode buttons cannot be focused upon

**Expected behavior**
Every element on the screen should receive focus as one swipes through the screen while using screen reading software.

**Screenshots**
See below. 

**Smartphone (please complete the following information):**
 - Device: Google Pixel
 - OS: Android
 - Version 2.5.8

<img width="418" alt="UnreachableButtons" src="https://user-images.githubusercontent.com/12497650/92796499-dac11980-f376-11ea-9c61-fb76ce394e5a.png">

**Comment by timo54** [comment]
This is a critical issue. It needs to be fixed as soon as possible.

**Comment by timo54** [comment]
This issue has been moved to the Safer Illinois repository and closed in this repository.
